scene/animation/tween.cpp

```diff
@@ -1203,9 +1216,16 @@ bool Tween::_calc_delta_val(const Variant &p_initial_val, const Variant &p_final
 			delta_val = Color(f.r - i.r, f.g - i.g, f.b - i.b, f.a - i.a);
 		} break;
 
+		case Variant::RECT2: {
+			// Build a new Rect2 and use the new position and sizes to make a delta
+			Rect2 i = initial_val;
+			Rect2 f = final_val;
+			delta_val = Rect2(f.position - i.position, f.size - i.size);
+		} break;
+
 		default:
 			// TODO: Should move away from a 'magic string'?
-			ERR_PRINT("Invalid param type, except(int/real/vector2/vector/matrix/matrix32/quat/aabb/transform/color)");
+			ERR_PRINT("Invalid param type, except(bool/int/real/Vector2/Vector/Basis/Transform2d/Quat/AABB/Transform/Color/Rect2)");
```

Comment:
`Transform2D` not `Transform2d`

Reply:
Maybe it's time to solve the TODO instead of keeping this hardcoded list... (And also fix the language, it should "expecting " not "except()".